server/beslog2json.py

```diff
@@ -72,6 +72,7 @@
 TRANSMIT_ERROR_LOG   = True
 TRANSMIT_VERBOSE_LOG = True
 TRANSMIT_TIMING_LOG  = False
+TRANSMIT_PROFILING_LOG  = True
```

Comment:
‼️ **Setting this to default true!!!**

Alternatively, we could set this to default false and set the flag directly in hyrax-docker/ngap/entrypoint.sh. Let's make sure we're on the same page about this choice before merging this branch!

Reply:
This is a nice minimally invasive way to go.